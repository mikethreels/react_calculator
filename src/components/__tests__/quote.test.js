import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../quote';

it('Quotes component renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
