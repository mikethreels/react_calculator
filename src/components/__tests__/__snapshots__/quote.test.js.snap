// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes component renders correctly 1`] = `
<div>
  <h1>
    Mathematics is not about numbers, equations, computations,
    <br />
    or algorithms: it is about understanding. â€“William Paul Thurston
  </h1>
</div>
`;
